lib/installer.js

```diff
@@ -105,6 +106,19 @@ function downloadRelease(version) {
         return yield tc.cacheDir(extPath, "protoc", version);
     });
 }
+function denormalizeVersion(version) {
+    const parsed = semver.parse(version);
```

Comment:
It is necessary to handle a `null` return value from `semver.parse` in the TypeScript code, otherwise the build will fail:

```text
src/installer.ts:122:7 - error TS2531: Object is possibly 'null'.

122   if (parsed.major > 3) {
          ~~~~~~

src/installer.ts:125:19 - error TS2531: Object is possibly 'null'.

125     version = `v${parsed.major}.${parsed.minor}`;
                      ~~~~~~

src/installer.ts:125:35 - error TS2531: Object is possibly 'null'.

125     version = `v${parsed.major}.${parsed.minor}`;
                                      ~~~~~~

src/installer.ts:126:32 - error TS2531: Object is possibly 'null'.

126     core.debug(`denormalized ${parsed.version} to ${version}`);
                                   ~~~~~~
```

I did this:

```typescript
  if (parsed === null) {
    throw new Error("unable to parse version as SemVer");
  }
```